api/api.go

```diff
@@ -14,12 +14,29 @@ const (
 	apiTimeout = 5 * time.Second
 )
 
+// HttpGET is a utility function for making http get requests with Sia-Agent set
+func HttpGET(url string) (resp *http.Response, err error) {
+	client := &http.Client{}
+	req, _ := http.NewRequest("GET", url, nil)
+	req.Header.Add("User-Agent", "Sia-Agent")
+	return client.Do(req)
+}
+
+// HttpPOST is a utility function for making post requests with Sia-Agent set
+func HttpPOST(url string, data string) (resp *http.Response, err error) {
+	client := &http.Client{}
+	req, _ := http.NewRequest("POST", url, strings.NewReader(data))
```

Comment:
sure, no harm in it.
Also, since we're not configuring the `client` at all, you can just use `return new(http.Client).Do(req)`.